compiler/src/dmd/parse.d

```diff
@@ -938,7 +938,8 @@ class Parser(AST, Lexer = dmd.lexer.Lexer) : Lexer
                     if (token.value == TOK.leftParenthesis)
                     {
                         nextToken();
-                        e = parseAssignExp();
+                        if (token.value == TOK.default_) nextToken();
+                        else e = parseAssignExp();
```

Comment:
the error message below for duplicate redundant `align` attributes below should possibly also be updated to print `align(default)` instead of `align`.

Reply:
Done